### PR TITLE
Get upgrades working, use kswap ns

### PR DESCRIPTION
**Pull request by slpopejoy**

### CODE REVIEW

File: pact/fungible-util.pact [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
-(namespace 'swap)
+(namespace (read-msg 'ns))
```
Comment (mightybyte) [inline pact/fungible-util.pact:2]: I like the use of `read-msg` all over the place to make the namespace DRY.

File: pact/fungible-util.pact [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 (module fungible-util GOVERNANCE
 
   (defcap GOVERNANCE ()
-    (enforce (keyset-ref-guard 'swap-ns-admin)))
+    (enforce-guard (keyset-ref-guard 'swap-ns-admin)))
```
Comment (mightybyte) [inline pact/fungible-util.pact:7]: Confirm that this is an important change.

File: pact/yaml/testnet/ns.yaml [hunk 3]
```diff
@@ -1,8 +1,10 @@
-codeFile: ../ns.pact
+codeFile: ../../ns.pact
 signers:
-  - public: dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e
+  - public: ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6
   - public: 368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca
 data:
+  ns: "kswap"
```
Comment (mightybyte) [inline pact/yaml/testnet/ns.yaml:6]: Looks good.

**Review by mightybyte — COMMENTED** [review]
Looks reasonable to me.  I'd like to get one more pair of eyes on it to verify (@ggobugi27 @fmelp).  Also, I didn't have the time to check whether we missed any uses of `enforce`.

File: pact/yaml/testnet/ns.yaml [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
+  upgrade: false
```
Comment (ggobugi27) [inline pact/yaml/testnet/ns.yaml:7]: `ns.pact` doesn't seem to use upgrade. Otherwise ns.yaml looks all good.

**Comment by ggobugi27** [comment]
Governance in `noop-callable` also needs change. 

https://github.com/kadena-io/kadenaswap/blob/b2d01a5c6316d4e67bb0fe61e9a16a9d2ddd32c7/pact/swap-callable.pact#L24

File: pact/exchange.repl [hunk 5]
```diff
@@ -22,10 +24,18 @@
 
 
 (define-namespace 'test (sig-keyset) (sig-keyset))
+(env-data { 'ns: "test" })
```
Comment (ggobugi27) [inline pact/exchange.repl:27]: Also needs `upgrade: false`

**Review by ggobugi27 — COMMENTED** [review]
Confirmed that ns change to `kswap` looks good. 
Caught governance fix for `swap-callable.pact`. 
Noted that `exchange.repl` needs one fix to add `upgrade` when loading test tokens.

File: pact/yaml/testnet/ns.yaml [hunk 6]
```diff
@@ -1,8 +1,9 @@
...
+codeFile: ../../ns.pact
 signers:
-  - public: dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e
+  - public: ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6
   - public: 368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca
 data:
+  ns: "kswap"
   swap-ns-user:
     keys:
       - dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e
```
Comment (LindaOrtega) [inline pact/yaml/testnet/ns.yaml:9]: I see that `dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e` was switched out with `ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6`, but I still see it in `keys`. Is that intentional?

File: pact/ns.pact [hunk 7]
```diff
@@ -1,8 +1,8 @@
 (define-keyset 'swap-ns-user)
 (define-keyset 'swap-ns-admin)
-(ns.write-registry 'swap (keyset-ref-guard 'swap-ns-admin) true)
+(ns.write-registry (read-msg 'ns) (keyset-ref-guard 'swap-ns-admin) true)
 (define-namespace
-  "swap"
+  (read-msg 'ns)
```
Comment (LindaOrtega) [inline pact/ns.pact:5]: Looks good!
`(read-msg 'ns)` would get evaluated to `kswap` based off `pact/yaml/testnet/ns.yaml`

**Review by LindaOrtega — COMMENTED** [review]
I took a look at `pact/ns.pact` and `pact/yaml/testnet/ns.yaml`. And left a small question. Looks good though!

File: pact/yaml/testnet/ns.yaml (hunk 6 above)
Comment (slpopejoy) [reply pact/yaml/testnet/ns.yaml:9]: Yes, this is the consequence of a different person (me) running the tx, but the keyset still having me and the previous runner (HK) in the keyset.

**Pull request merged**